### PR TITLE
Order ladder UI

### PR DESCRIPTION
**Pull request by alexmatson-da**
Screen Shot 2021-01-13 at 9 13 47 PM

### CODE REVIEW

File: ui/src/index.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@
   --spacing-xs: 6px;
   --spacing-s: 12px;
   --spacing-m: 24px;
+  --spacing-ml: 32px;
```
Comment (alexmatson-da) [inline ui/src/index.scss:13]: "medium large"? heh, didn't know what to call this... open to suggestions

Comment (alexgraham-da) [reply ui/src/index.scss:13]: "venti" ☕

**Review by alexgraham-da — APPROVED** [review]
looks great!

File: ui/src/index.scss (hunk 1 above)
Comment (Sarahb-da) [reply ui/src/index.scss:13]: pissbily change all the `spacing-l` to `spacing-xl`:`48px` and make that the new `spacing-l`

**Review by Sarahb-da — APPROVED** [review]
lgtm!

**Pull request merged**